ext/rp/module.lb

```diff
@@ -44,12 +44,20 @@ def build(env):
     env.copy("pico-sdk/include",'')
     env.copy("address_mapped.h",'hardware/address_mapped.h')
 
+    flash_variant = env.get("boot2")
+    env.copy("pico-sdk/src/boot2_" + flash_variant + '.cpp','src/boot2.cpp')
```

Comment:
I think the option and the copy code of boot2 should stay in `:platform:core`, it makes more sense there since that module is "closer" to the startup than the "passive" CMSIS if that makes sense.